cmd/talosctl/cmd/mgmt/cluster/create/cmd.go

```diff
@@ -131,19 +132,21 @@ func addTalosconfigDestinationFlag(flagset *pflag.FlagSet, bind *string, flagNam
 }
 
 func addControlplaneCpusFlag(flagset *pflag.FlagSet, bind *string, flagName string) {
-	flagset.StringVar(bind, flagName, "2.0", "the share of CPUs as fraction (each control plane/VM)")
+	flagset.StringVar(bind, flagName, "2.0", "the share of CPUs as fraction for each control plane/VM")
 }
 
 func addWorkersCpusFlag(flagset *pflag.FlagSet, bind *string, flagName string) {
-	flagset.StringVar(bind, flagName, "2.0", "the share of CPUs as fraction (each worker/VM)")
+	flagset.StringVar(bind, flagName, "2.0", "the share of CPUs as fraction for each worker/VM")
 }
 
-func addControlPlaneMemoryFlag(flagset *pflag.FlagSet, bind *int, flagName string) {
-	flagset.IntVar(bind, flagName, 2048, "the limit on memory usage in MB (each control plane/VM)")
+func addControlPlaneMemoryFlag(flagset *pflag.FlagSet, bind *bytesize.ByteSize, flagName string) {
+	cli.Should(bind.Set("2.0gb")) // set default value
+	flagset.Var(bind, flagName, "the limit on memory usage for each control plane/VM")
 }
 
-func addWorkersMemoryFlag(flagset *pflag.FlagSet, bind *int, flagName string) {
-	flagset.IntVar(bind, flagName, 2048, "the limit on memory usage in MB (each worker/VM)")
+func addWorkersMemoryFlag(flagset *pflag.FlagSet, bind *bytesize.ByteSize, flagName string) {
+	cli.Should(bind.Set("2.0gb")) // set default value
```

Comment:
I think this should be 
```suggestion
	cli.Should(bind.Set("2.0GiB")) // set default value
```